packages/math-input/src/types.ts

```diff
@@ -68,3 +68,69 @@ export type KeypadContextType = {
     ) => void;
     scrollableElement: HTMLElement | null | undefined;
 };
+
+/**
+ * The translated strings that are used to render the Math Input.
+ */
+export type MathInputStrings = {
+    mathInputBox: string;
+    fingerTap: string;
+    before: ({obj}: {obj: string}) => string;
+    after: ({obj}: {obj: string}) => string;
+    "beginning of": ({obj}: {obj: string}) => string;
+    "end of": ({obj}: {obj: string}) => string;
+    Baseline: string;
+    Superscript: string;
+    selected: ({obj}: {obj: string}) => string;
+    "no answer": string;
+    "nothing selected": string;
+    "nothing to the right": string;
+    "nothing to the left": string;
+    "block is empty": string;
+    "nothing above": string;
+    labelValue: ({label, value}: {label: string; value: string}) => string;
+    plus: string;
+    minus: string;
+    negative: string;
+    multiply: string;
+    divide: string;
+    decimal: string;
+    percent: string;
+    multiply2: string;
```

Comment:
I feel it might be a little strange having a string called multiply2. I wonder if it makes sense to name them strings.cdot and strings.times, and just have the resulting string be "Multiply" for both? It might help make it a little easier to understand in key-config. 

This is a total nit though, and not something I consider an issue.

Reply:
Ok, yeah - I can change that!